examples/common.h

```diff
@@ -32,6 +32,7 @@ struct gpt_params {
     int32_t main_gpu                        = 0;   // the GPU that is used for scratch and small tensors
     float   tensor_split[LLAMA_MAX_DEVICES] = {0}; // how split tensors should be distributed across GPUs
     bool    low_vram                        = 0;   // if true, reduce VRAM usage at the cost of performance
```

Comment:
Minor nit : this `bool` should be together with the other `bool`s in the struct

Reply:
Just changed. It doesn;t really fit in the topic of this pr tho.